apps/api/src/app/environments-v1/usecases/output-renderers/email-output-renderer.usecase.ts

```diff
@@ -262,6 +268,14 @@ export class EmailOutputRendererUsecase {
         return processedNode;
       }
 
+      if (isButtonNode(processedNode)) {
```

Comment:
the array item access in the button that is inside the repeat block was not working